pages/api/contrataciones.js

```diff
@@ -0,0 +1,81 @@
+import puppeteer from "puppeteer";
+
+function separeCuseId(id) {
```

Comment:
please try to use arrow function : `const separeCuseId = (id) => {`
also change the name to parseCuse